specification/sql/resource-manager/readme.md

```diff
@@ -716,16 +711,107 @@ These can be regenerated by running the following PowerShell script from this re
 
 ``` yaml $(tag) == 'package-pure-2020-02-preview'
 input-file:
-- ./Microsoft.Sql/preview/2020-02-02-preview/DatabaseSecurityAlertPolicies.json
-- ./Microsoft.Sql/preview/2020-02-02-preview/ManagedServerSecurityAlertPolicies.json
-- ./Microsoft.Sql/preview/2020-02-02-preview/ServerSecurityAlertPolicies.json
-- ./Microsoft.Sql/preview/2020-02-02-preview/operations.json
-- ./Microsoft.Sql/preview/2020-02-02-preview/managedDatabases.json
-- ./Microsoft.Sql/preview/2020-02-02-preview/managedInstances.json
-- ./Microsoft.Sql/preview/2020-02-02-preview/ServerAzureADOnlyAuthentications.json
-- Microsoft.Sql/preview/2020-02-02-preview/importexport.json
-- ./Microsoft.Sql/preview/2020-02-02-preview/ManagedInstanceAzureADOnlyAuthentications.json
-- ./Microsoft.Sql/preview/2020-02-02-preview/ServerTrustGroups.json
+ - ./Microsoft.Sql/preview/2020-02-02-preview/BackupShortTermRetentionPolicies.json
```

Comment:
All the newly added json files are carried from previous versions. They are NOT the new introduced API endpoints.